src/agent/src/process_options.cpp

```diff
@@ -7,6 +7,27 @@
 void StartAgent()
 {
     LogInfo("Starting Wazuh Agent.");
+
+    if (AgentDaemon::IsAlreadyRunning())
+    {
+        LogInfo("Wazuh-Daemon is already running");
+        return;
+    }
+
+    int ret = AgentDaemon::Daemonize();
+
+    if (ret == 0)
+    {
+        LogInfo("Wazuh-Daemon started");
```

Comment:
Nitpick, could we say "Wazuh-Daemon starting..." ?

It could still fail to start between this log file and the actual call to `agent.Run()` starts the event loop.

Reply:
Done